source/posix/socket.c

```diff
@@ -533,6 +534,11 @@ int aws_socket_connect(
         AWS_ASSERT(on_connection_result);
     }
 
+    size_t address_strlen;
+    if (aws_secure_strlen(remote_endpoint->address, AWS_ADDRESS_MAX_LEN, &address_strlen)) {
```

Comment:
yaay! somebody finally using this function.

Reply:
We should support passing NULL as the out-param. I just wanted to sanity check that the string was long enough